### PR TITLE
Multiple mss

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Corrections,  LIT5069Prayer and LIT6205Saalnaka

### CODE REVIEW

File: DayrSuryan/DSEthiop9.xml [hunk 1]
```diff
@@ -58,9 +58,9 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         <textLang mainLang="gez"/>
 
                      <incipit xml:lang="gez" type="inscriptio">
-                         <hi rend="rubric">በስመ፡ አብ ወወልድ፡ ወመንፈ<sic resp="DR">ን</sic>ስ፡ ቅዱስ፡</hi>
+                         <hi rend="rubric">በስመ፡ አብ ወወልድ፡ <sic resp="DR">ወመንፈንስ፡</sic> ቅዱስ፡</hi>
```
Comment (thea-m) [inline DayrSuryan/DSEthiop9.xml:61]: I think that this change has been carried out to change the visualisation. @PietroLiuzzo the fact that `<sic>` is not yet rendered as (!) after the word (not the letter) does not mean that it is possible, right?

**Review by thea-m — CHANGES_REQUESTED** [review]

**Comment by DenisNosnitsin1970** [comment]
Can this be merged?

File: DayrSuryan/DSEthiop9.xml (hunk 1 above)
Comment (thea-m) [reply DayrSuryan/DSEthiop9.xml:61]: @PietroLiuzzo could you confirm (or correct me)?

**Comment by thea-m** [comment]
I still think that changing the exact markup of sic is not necessary, let's wait for @PietroLiuzzo 's answer

**Comment by DenisNosnitsin1970** [comment]
Dorothea, let us merge it?

**Review by thea-m — APPROVED** [review]

**Pull request merged**